editor.planx.uk/src/pages/FlowEditor/lib/analytics/utils.ts

```diff
@@ -190,3 +190,50 @@ export function getAllowListAnswers(
 
   return allowListAnswers;
 }
+
+export type Environment = keyof typeof DASHBOARD_PUBLIC_IDS;
+
+// The dashboard links across Metabase staging and production are different, so we need to store and be able to access both
+export const DASHBOARD_PUBLIC_IDS = {
+  production: {
+    discretionary: "34e5a17c-2f20-4543-be0b-4af8364dceee",
+    FOIYNPP: "d55b0362-3a21-4153-a53d-c1d6a55ff5e2",
+    RAB: "f3da39ec-bb4f-4ee0-8950-afcb5765d686",
+    submission: "040dad13-6783-4e48-9edc-be1b03aa5247",
+  },
+  staging: {
+    discretionary: "0c0abafd-e919-4da2-a5b3-1c637f703954",
+    FOIYNPP: "d6303f0b-d6e8-4169-93c0-f988a93e19bc",
+    RAB: "85c120bf-39b0-4396-bf8a-254b885e77f5",
+    submission: "363fd552-8c2b-40d9-8b7a-21634ec182cc",
+  },
+} as const;
+
+export const getFOIYNPP = (environment: Environment) => ({
+  id: DASHBOARD_PUBLIC_IDS[environment].FOIYNPP,
+  slugs: [
+    "check-if-you-need-planning-permission",
+    "find-out-if-you-need-planning-permission",
+  ],
+});
+export const getRAB = (environment: Environment) => ({
+  id: DASHBOARD_PUBLIC_IDS[environment].RAB,
+  slugs: ["report-a-planning-breach"],
+});
+export const getSubmission = (environment: Environment) => ({
+  id: DASHBOARD_PUBLIC_IDS[environment].submission,
+  slugs: [
+    "apply-for-planning-permission",
+    "apply-for-a-lawful-development-certificate",
+    "pre-application",
+  ],
+});
+export const getDiscretionary = (environment: Environment) => ({
```

Comment:
```suggestion
});

export const getRAB = (environment: Environment) => ({
  id: DASHBOARD_PUBLIC_IDS[environment].RAB,
  slugs: ["report-a-planning-breach"],
});

export const getSubmission = (environment: Environment) => ({
  id: DASHBOARD_PUBLIC_IDS[environment].submission,
  slugs: [
    "apply-for-planning-permission",
    "apply-for-a-lawful-development-certificate",
    "pre-application",
  ],
});

export const getDiscretionary = (environment: Environment) => ({
```

nit: A bit of whitespace here really helps with readability.